pkg/visitor/list.go

```diff
@@ -40,6 +43,13 @@ func ListProjects(ctx context.Context,
 		Filter: filter,
 	})
 	for r, err := it.Next(); err != iterator.Done; r, err = it.Next() {
+		if err != nil && status.Code(err) == codes.Unimplemented && implicitProject {
+			// If the admin service is unavailable, provide a placeholder project.
+			// If the project is invalid, downstream actions will fail.
+			p := &rpc.Project{Name: name.String()}
```

Comment:
Questionable naming: This would likely fail validation, but the other option would be to hard-code a name. Thoughts?

Reply:
Could we pass in the project name instead of a bool?

Reply:
Good idea!